src/en/experts-sla.md

```diff
@@ -1,149 +1,257 @@
-Title: Managed solutions under SLA
+Title: Managed solutions
```

Comment:
Should this be Juju Experts as that's the branding of the offering?

Reply:
I don't think so. The software allows for "managed solutions". It does not manage the Juju Experts program.